queue/src/fiskaltrust.Middleware.Localization.QueueDE/SignatureTypesDE.cs

```diff
@@ -23,6 +23,7 @@ public enum SignatureTypesDE
         VorgangsBeginn = 0x1F,
         UpdateTransactionPayload = 0x20,
         UpdateTransactionResult = 0x21,
-        CertificationId = 0x22
+        CertificationId = 0x22,
+        TseSerialNumber = 0x23
```

Comment:
@mijomilicevic @saschaknapp just wanted to hear your opinion on that one. IMO this should be a own signaturetype but just wanted to check in with you. If we keep that we also will need to extend interface-doc

Reply:
Having a new signaturetype for this makes sense in my opinion too :)

Reply:
@paulcristiann can you maybe extend the list at https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/germany/reference-tables/ftsignaturetype ? Should also be backed by github :)

Reply:
Sure, on it

Reply:
@StefanKert We should be good to go